src/services/Content/ContentApi.ts

```diff
@@ -330,12 +330,30 @@ export const introspectRepository: (
   return data;
 };
 
+export const triggerSnapshot: (repositoryUUID: string) => Promise<void> = async (
+  repositoryUUID,
+) => {
+  try {
+    const { data } = await axios.post(
+      `/api/content_sources/v1.0/repositories/${repositoryUUID}/snapshot/`,
+      {},
+    );
+    return data;
+  } catch (error: unknown) {
```

Comment:
This function looks good now!  (aside from the error handling, but we'll handle that once its fully working)